src/explorer-components/BountyPayoutData/BountyPayoutData.module.scss

```diff
@@ -0,0 +1,10 @@
+.payoutEthBox {
+    margin-top: 0.5rem;
+}
+.balanceEthBox {
+    margin-top: 1.5rem;
+    margin-bottom: 0.5rem;
+}
+.currencyTypeHeaderText {
+    font-size: 0.775rem;
```

Comment:
Same here, let's try and use value's based on our typographic scale. If we can't use a prop, check the variables file for the typographic scale values.